crates/mpz-zk-core/src/vole.rs

```diff
@@ -0,0 +1,65 @@
+//! Vector oblivious polynomial evaluation (VOPE) protocol for degree 1.
+//!
+//! This implementation is specifically for degree 1, i.e. VOLE over the
+//! extension field.
```

Comment:
I think you meant VOPE in this file? 
This implementation simplies figure 4 for a polynomial with degree 1.

Reply:
It is different. For boolean circuits, VOLE is Random COT. VOLE is a generalized version to extend boolean to arithmetic circuits.
VOPE focuses on polynomial relations. B = A_0 + A_1*Delta +..., but all the B and A_i are in F_2^{128}.